src/components/InstancesList/InstancesList.js

```diff
@@ -1077,39 +1090,53 @@ class InstancesList extends React.Component {
         title,
         discoverySuppress,
         isBoundWith,
+        shared,
```

Comment:
Where does this property come from?

Reply:
From `contentData` array. It's a new property of `instance`

Reply:
Good to know, thank you @BogdanDenis !